internal/app/parcel_carrier/repository_test.go

```diff
@@ -0,0 +1,88 @@
+package parcel_carrier
+
+import (
+	"context"
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"parcel-service/internal/app/model"
+	"testing"
+	"time"
+)
+
+func TestRepository_UpdateCarrierRequest(t *testing.T) {
+	var sourceTime = time.Now()
+	t.Run("should return success", func(t *testing.T) {
+		db, m, _ := sqlmock.New()
+		defer db.Close()
+
+		parcel := model.CarrierRequest{
+			ParcelID: 1,
+			CarrierID:    2,
+			Status:  1,
+		}
+
+		sqlxDB := sqlx.NewDb(db, "sqlmock")
+		m.ExpectBegin()
+		m.ExpectExec("UPDATE carrier_request SET (.+) WHERE (.+) AND (.+)").
+			WithArgs(2, parcel.ParcelID, parcel.CarrierID).
+			WillReturnResult(sqlmock.NewResult(1, 1))
+		m.ExpectExec("UPDATE carrier_request SET (.+) WHERE (.+) AND (.+)").
+			WithArgs(3, parcel.ParcelID, parcel.CarrierID).
+			WillReturnResult(sqlmock.NewResult(1, 1))
+		m.ExpectExec("UPDATE parcel SET (.+) WHERE (.+)").
+			WithArgs(parcel.CarrierID, 2, sourceTime, parcel.ParcelID).
+			WillReturnResult(sqlmock.NewResult(1, 1))
+		m.ExpectCommit()
+
+		repo := NewRepository(sqlxDB)
+		err := repo.UpdateCarrierRequest(context.Background(), parcel, 2, 3, 2, sourceTime)
+		assert.True(t, errors.Is(err, nil))
```

Comment:
use `nil` assert

Reply:
Why to use assert `nil`? When i use `nil`.It throws an status code error with exit 1.